src/commands/apps/create.ts

```diff
@@ -18,25 +18,44 @@ export const appCreateCommand: CommandModule<
 		token: { type: "string", demandOption: true, hidden: true },
 	},
 	handler: async (args) => {
-		const client = getApiClient(args.token);
 		try {
-			const response = await client.createApp({
-				createAppRequest: {
+			const response = await client.apps.create(
+				{
 					appName: args.appName,
 					authConfiguration: {
 						anonymous: {},
 					},
 				},
-			});
-			console.log(response);
+				{
+					auth0: `Bearer ${args.token}`,
+				}
+			);
+			switch (response.statusCode) {
+				case 201:
+					console.log(response.application);
+					break;
+				case 422:
+					ERROR_MESSAGES.RESPONSE_ERROR(
+						response.statusCode.toString(),
+						response.createApp422ApplicationJSONString
```

Comment:
These are ugly names, but we could modify the schema to create better ones in the SDK

Reply:
We have a custom extension to do this and are using it for method names in your spec. But we can also apply it to Speakeasy-generated class names: https://speakeasyapi.dev/docs/using-speakeasy/create-client-sdks/customize-sdks/methods/